tools/CreateMappings.ps1

```diff
@@ -57,5 +57,9 @@ $results | ConvertTo-Json | Out-File $OutputFile;
 # Print conclusion.
```

Comment:
@twitchax as a side note: since this cmdlet is writing to the progress stream (👎), we should turn it off during CI so it doesn't slow the build down like it did for help generation. Currently it's take ~40 seconds for this script to run, so we could shave off some time if we set `ProgressPreference = 'SilentlyContinue'` for this script execution.

[Here's how we do it for `GenerateHelp.ps1`](https://github.com/Azure/azure-powershell/blob/preview/build.proj#L207).

Reply:
Good point: will do.